tests/Dfc.CourseDirectory.Testing/TestData/TestData.CreateTLevelDefinition.cs

```diff
@@ -20,5 +22,59 @@ public async Task<Guid> CreateTLevelDefinition(Guid? tLevelDefinitionId = null,
 
             return id;
         }
+
+        public Task<IReadOnlyCollection<TLevelDefinition>> CreateInitialTLevelDefinitions() =>
+            WithSqlQueryDispatcher(async dispatcher =>
+            {
+                var createCommands = new[]
+                {
+                    new CreateTLevelDefinition()
+                    {
+                        TLevelDefinitionId = new Guid("9555d14f-f73b-495c-91c3-4703691c6347"),
+                        FrameworkCode = 36,
+                        ProgType = 32,
+                        Name = "T Level Digital - Digital Production, Design and Development",
```

Comment:
Do we need to include such verbose test data?

Reply:
I thought something realistic was better than just duplicating the property name.